virtwho/virt/ahv/ahv.py

```diff
@@ -114,6 +114,8 @@ def getHostGuestMapping(self):
         mapping = {'hypervisors': []}
 
         host_uvm_map = self._interface.build_host_to_uvm_map(self.version)
+        if host_uvm_map:
+            cluster_ids = self._interface.get_ahv_cluster_uuid_map(self.version)
```

Comment:
I would rename this method to `cluster_uuid_map` to emphasize that it is map of uuids.